dotCMS/src/main/java/com/dotcms/rest/ResourceResponse.java

```diff
@@ -82,11 +83,18 @@ public Response response () {
      * @param response String with the data to response, this response format should depend if exist of the <strong>"type"</strong> parameter
      * @return
      */
-    public Response response ( String response ) {
+    public Response response ( final String response ) {
         return response( response, null );
     }
 
-    public Response response ( String response, CacheControl cacheControl ) {
+    public Response response ( final String response, final CacheControl cacheControl ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "cacheControl". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)